### ISSUE TITLE
`lnp-cli create` hangs after channel creation

### ISSUE THREAD

**Issue by St333p** [LNP-BP/lnp-node#25]
**Checklist**
- [ ] Make sure you are using the latest code (`git checkout master && git pull origin`)
- [x] Compile only with rust nightly: `rustup default nightly && rustup update nightly && cargo test --all --all-features -v`
- [x] Try to do `cargo update`
- [x] Try to remove `target` directory

**Classify the bug**
Put `x` in the boxes below:
- [ ] Build issue
- [ ] Test failing
- [ ] Runtime panic
- [ ] Incorrect results
- [x] Unexpected/undocumented behaviour / 

**Describe the problem**
In the latest tag (`v0.1.0-alpha.4`, at 4f14b5b), a call to `lnp-cli create` hangs forever after channel creation.

I assume channels get created before the issue arises because of the following log:
```console=
[2020-11-04T17:59:58Z INFO  lnp_node::lnpd::runtime] Channel daemon b0459cab9aefa51153067966471c11aa5f6d90ba5233b3b514f0293ff2d22ce2 is registered; total 3 channels are known
```
and because the `channels` field in `lnp-cli info` increases correctly.

However `lnp-cli channels` still returns an empty string.

**To Reproduce**
1. Operating system
Debian buster
2. Rust compiler version
1.47.0
3. Did you 
    - I set up  node in listening mode `lnp-cli listen`
    - I connected a second node successfully with `lnp-cli connect`
    - I used `lnp-cli create` to create a channel between the two nodes
 
**Expected behavior**
The command line call to `lnp-cli create` returns after channel creation

**Comment by dr-orlovsky** [comment]
Because the channel creation was not finished at that alpha :) It's completed with the next version I used for the demo yesterday

**Comment by St333p** [comment]
This is fixed since tag v0.1.0-beta.1.
